.vscode/launch.json

```diff
@@ -0,0 +1,36 @@
+{
+    // Use IntelliSense to find out which attributes exist for C# debugging
+    // Use hover for the description of the existing attributes
+    // For further information visit https://github.com/OmniSharp/omnisharp-vscode/blob/master/debugger-launchjson.md
+    "version": "0.2.0",
+    "configurations": [
+        {
+			"name": "Go Launch",
```

Comment:
This enables us to debug go code. I have tested this and it works fine. Also, will fix the spacings here